mathoms.c

```diff
@@ -219,18 +219,6 @@ Perl_sv_force_normal(pTHX_ SV *sv)
     sv_force_normal_flags(sv, 0);
 }
 
-/* sv_setsv() is now a macro using Perl_sv_setsv_flags();
- * this function provided for binary compatibility only
- */
-
-void
-Perl_sv_setsv(pTHX_ SV *dsv, SV *ssv)
-{
-    PERL_ARGS_ASSERT_SV_SETSV;
-
-    sv_setsv_flags(dsv, ssv, SV_GMAGIC);
-}
```

Comment:
This mathom.c linker symbol stub's .c code, DOES NOT match the macro. Touching this sv.c/sv.h code logic  absolutely shouldn't be part of a generic "improve embed.fnc for easier use" PR/commit, and needs to be done in a separate PR/commit.
 
```
#define sv_setsv(dsv, ssv) \
        sv_setsv_flags(dsv, ssv, SV_GMAGIC|SV_DO_COW_SVSETSV)
``

Reply:
You're right.  I forgot that the mathoms.c code for this function and the sv.h definition had gotten out of sync.  I changed to use a different function that doesn't have an inconsistency